__tests__/index.test.js

```diff
@@ -374,4 +374,39 @@ describe('DELETE /api/comments/:comment_id', () => {
         expect(body.msg).toEqual(expected);
       });
   });
+});
+describe('GET /api/users', () => {
+  it('GET 200: should respond with an array of objects, each object should have the following properties: username, name, avatar_url', () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const { users } = body;
+        const length = users.length;
+
+        expect(length).toBe(4);
+
+        users.forEach((user) => {
+          expect(Object.keys(user).length).toBe(3);
+          expect(Object.keys(user)).toEqual(['username', 'name', 'avatar_url']);
+          const { username, name, avatar_url} =
+            user;
+          expect(typeof username).toBe("string");
+          expect(typeof name).toBe("string");
+          expect(typeof avatar_url).toBe("string");
```

Comment:
These five expect statements could be replaced with one `.toMatchObject` expect statement 🔍